### PR TITLE
Add production terraform for dms

### PR DESCRIPTION
**Pull request by MirelaGeorgiou**

### CODE REVIEW

**Review by humulla — APPROVED** [review]

File: terraform/production/main.tf [hunk 1]
```diff
@@ -64,4 +64,44 @@ module "postgres_db_production" {
...
+data "aws_ssm_parameter" "academy_username" {
+   name = "/academy/reporting-server/username"
+}
+data "aws_ssm_parameter" "academy_password" {
+   name = "/academy/reporting-server/password"
+}
+data "aws_ssm_parameter" "academy_hostname" {
+   name = "/academy/reporting-server/hostname"
+}
+module "dms_setup_staging" {
```
Comment (elcorbs) [inline terraform/production/main.tf]: Should this be production?

File: terraform/production/task_settings.json [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+        "TargetTablePrepMode": "DROP_AND_CREATE",
+        "CreatePkAfterFullLoad": false,
+        "StopTaskCachedChangesApplied": false,
+        "StopTaskCachedChangesNotApplied": false,
+        "MaxFullLoadSubTasks": 8,
+        "TransactionConsistencyTimeout": 600,
+        "CommitRate": 10000
+    },
+    "Logging": {
+        "EnableLogging": false,
```
Comment (elcorbs) [inline terraform/production/task_settings.json]: As this is production, should we enable logging here?

File: terraform/production/main.tf (hunk 1 above)
Comment (MirelaGeorgiou) [reply terraform/production/main.tf]: I missed that, thanks for pointing it out

File: terraform/production/task_settings.json (hunk 2 above)
Comment (MirelaGeorgiou) [reply terraform/production/task_settings.json]: Yep, missed that too :) Thanks!

**Pull request merged**